lib/jekyll/commands/new.rb

```diff
@@ -27,8 +27,8 @@ def process(args, options = {})
           new_blog_path = File.expand_path(args.join(" "), Dir.pwd)
           FileUtils.mkdir_p new_blog_path
           if preserve_source_location?(new_blog_path, options)
-            Jekyll.logger.abort_with "Conflict:",
-                      "#{new_blog_path} exists and is not empty."
+            Jekyll.logger.error "Conflict:", "#{new_blog_path} exists and is not empty."
+            Jekyll.logger.abort_with "", "Use '--force' to overwrite and create anyway."
```

Comment:
I'm not sure about this wording. Can we make it a bit less forceful? Using `--force` shouldn't be recommended as the default option.

Reply:
> bit less forceful?

haha..


> Using `--force` shouldn't be recommended as the default option.

```
"Ensure the #{new_blog_path} is empty or else try again with `--force`
to proceed and overwrite any files"
```